src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,46 @@
+package roomescape.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.domain.Reservation;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private final List<Reservation> reservations = new ArrayList<>();
+    private final AtomicLong index = new AtomicLong(1);
+
+    @GetMapping
+    public ResponseEntity<List<FindReservationResponse>> getReservation() {
+        List<FindReservationResponse> reservationsDtos = reservations.stream()
+                .map(FindReservationResponse::of)
+                .toList();
```

Comment:
[제안]
개인적으로 dto라는 표현보다는 responses가 어떨까 싶어요.

```suggestion
    public ResponseEntity<List<FindReservationResponse>> getReservation() {
        List<FindReservationResponse> reservationsResponses = reservations.stream()
                .map(FindReservationResponse::of)
                .toList();
```

Reply:
앗 dto 이름을 변경하면서 놓친 부분이 있었네요🥲
반영하겠습니다! 🙇